packages/ui/src/molecules/PaymentMethods/PaymentMethods.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import Label from '../../atoms/Label'
+
+export interface PaymentMethodsProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Title of the payment methods section (e.g.: "Payment methods",
+   * "Accepted Cards").
+   */
+  title?: ReactNode
+  /**
+   * Children will receive the flags to be displayed in the payment
+   * methods section (e.g.:, visa, mastercard, etc).
+   */
+  children: ReactNode
+}
+
+const PaymentMethods = ({
```

Comment:
I suggest including that usual `forwardRef` mechanic you can find in other components (e.g. in [Label](https://github.com/vtex/faststore/blob/492088aa1d9a3eb6cf2f8a159c513f66d12f9122/packages/ui/src/atoms/Label/Label.tsx#L11)) to keep the pattern going.